www/src/js/components/search-field.js

```diff
@@ -35,5 +35,6 @@ module.exports = React.createClass({
       </div>
     );
   }
-})
+}
 
+module.exports = SearchField;
```

Comment:
In theory yes, but it doesn't seem to like it. It may be an either/or choice for module type.